modules/geo-layers/src/mesh-layer/mesh-layer.js

```diff
@@ -0,0 +1,146 @@
+import {GLTFMaterialParser} from '@luma.gl/experimental';
+import {Model, Geometry, pbr} from '@luma.gl/core';
+import {log} from '@deck.gl/core';
+import {SimpleMeshLayer} from '@deck.gl/mesh-layers';
+
+import vs from './mesh-layer-vertex.glsl';
+import fs from './mesh-layer-fragment.glsl';
+
+function validateGeometryAttributes(attributes) {
+  const hasColorAttribute = attributes.COLOR_0 || attributes.colors;
+  if (!hasColorAttribute) {
+    attributes.colors = {constant: true, value: new Float32Array([1, 1, 1])};
+  }
+  log.assert(
+    attributes.positions || attributes.POSITION,
+    'no "postions" or "POSITION" attribute in mesh'
+  );
+}
+
+/*
+ * Convert mesh data into geometry
+ * @returns {Geometry} geometry
+ */
+function getGeometry(data) {
```

Comment:
Simply this function since you control the mesh format that is passed to this layer.

Reply:
Done